docs/application/native/guides/ui/tizen-ws-shell/tzsh-quickpanel.md

```diff
@@ -0,0 +1,125 @@
+For example, media player application needs to close the Quickpanel window during playback of video. In this case, you can use the TZSH-Quickpanel library.
+
+## Prerequisites
+
+To use the functions and structures of the TZSH-Quickpanel API (in [mobile](../../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) and [wearable](../../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html) applications), include the `<tzsh_quickpanel.h>` header file in your application:
+
+```
+#include <tzsh_quickpanel.h>
+```
+
+## Create TZSH-Quickpanel Handler
+After you have created the main window of your application, call `tzsh_quickpanel_create_with_type` with a native window ID to create the `tzsh_quickpanel_h` structure (in [mobile](../../../api/mobile/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) and [wearable](../../../api/wearable/latest/group__TIZEN__WS__SHELL__QUICKPANEL__MODULE.html#gaaa00e8e25b43c9538ca188bc43bdb3ac) applications).
+
+```
+#include <Elementary.h>
+
+static void
+init(const char *name)
+{
+   Evas_Object *main_win;
+
+   /* create main window for the application */
+   main_win = elm_win_util_standard_add(name, name);
+
+   /* set up main window */
+   ...
+   evas_object_show(main_win);
+
+
+   tzsh_h tzsh;
+   tzsh_quickpanel_h tzsh_qp;
+   tzsh_window tz_win;
+
+   /* Get native window ID of main window */
+   tz_win = elm_win_window_id_get(main_win);
+
+   /* Create tzsh_h structure */
+   tzsh = tzsh_create(TZSH_TOOLKIT_TYPE_EFL);
+
+   /* Create tzsh_quickpanel_h structure */
+   tzsh_qp = tzsh_quickpanel_create_with_type(tzsh, tz_win, TZSH_QUICKPANEL_TYPE_SYSTEM_DEFAULT);
+```
+
+## Show or Hide Quickpanel Window
+
+To show or hide Quickpanel window when your application's window is activated, call the following functions:
+
+```
+   if (show)
+     tzsh_quickpanel_show(tzsh_qp); /* To show quickpanel window */
+   else
+     tzsh_quickpanel_hide(tzsh_qp); /* To hide quickpanel window */
+```
+
+## Get Visibility Status of Quickpanel Window
+To know the state of current visibility of Quickpanel service window, call the `tzsh_quickpanel_visible_get` function. The Quickpanel service window may be visible or invisible on the screen.
+
+```
+   tzsh_quickpanel_state_visible_e state;
+
+   tzsh_quickpanel_visible_get(tzsh_qp, &state);
+   if (state == TZSH_QUICKPANEL_STATE_VISIBLE_SHOWN) {
+     /* visible state */
+   } else if (state == TZSH_QUICKPANEL_STATE_VISIBLE_HIDDEN) {
+     /* invisible state */
+   } else {
+     /* error */
+   }
+
+```
+
+## Register a Changed Event for Quickpanel Window
+To be notified about the state changes, implement the appropriate event callback function and call the `tzsh_quickpanel_event_handler_add` function with that. If you want to change your application's behavior to match visibility of the quicknapel service window, you need to handle state change event as follows:
```

Comment:
Please change the spelling "quicknapel" to "Quickpanel".
Can we rephrase to:
To get notified about the state changes, implement the appropriate event callback function and call `tzsh_quickpanel_event_handler_add` with that. If you want to change your application's behavior to match the visibility of the Quickpanel service window, you need to handle the state change events as follows:

Question:
To get notified about the state changes, implement the appropriate event callback function and call `tzsh_quickpanel_event_handler_add` with that.

Here, "that" refers to "appropriate event callback function"? If it is, then can we clarify it in the sentence?
For more information, https://github.com/Samsung/tizen-docs/pull/748#discussion_r296112244.

Reply:
It isn't under scope to change, but I already modified other part of this article.
So I modified as your comment.

Answer: Yes, "that" means "event callback function"
How about rephrase to:
To get notified about the state changes, implement the appropriate event callback function and call tzsh_quickpanel_event_handler_add with that event callback function.

Reply:
The suggested rephrase clarifies better. Looks good.

Reply:
Thanks, I modified it